src/declaration.d

```diff
@@ -653,19 +653,22 @@ extern (C++) final class AliasDeclaration : Declaration
             type = null;
             aliassym = s;
         }
+        inuse = 0;
         if (global.gag && errors != global.errors)
         {
             type = oldtype;
             aliassym = null;
         }
-        inuse = 0;
-        semanticRun = PASSsemanticdone;
-
-        if (auto sx = overnext)
+        else
         {
-            overnext = null;
-            if (!overloadInsert(sx))
-                ScopeDsymbol.multiplyDefined(Loc(), sx, this);
+            semanticRun = PASSsemanticdone;
```

Comment:
The change of src/declaration.d seems unrelated to the segfault fix, can we remove it?
If you intended to rerun semantic to produce the right error message, that apparently didn't work anyhow.